### PR TITLE
added loki://?id deeplink

### PR DESCRIPTION
**Pull request by privatgt**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -139,7 +139,15 @@
...
             android:screenOrientation="portrait"
             android:windowSoftInputMode="adjustResize"
-            android:theme="@style/Theme.Session.DayNight.FlatActionBar" />
+                  android:theme="@style/Theme.Session.DayNight.FlatActionBar">
+        
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="loki"/>
```
Comment (jagerman) [inline app/src/main/AndroidManifest.xml:148]: Seems like this should probably be `session` rather than `loki`

Comment (privatgt) [reply app/src/main/AndroidManifest.xml:148]: `session` already in use, that's why I used `loki`

Comment (hjubb) [reply app/src/main/AndroidManifest.xml:148]: I think we will have to synchronise this across the iOS and Desktop platforms, but `loki` might not be an ideal fit. We may be able to use a combination of either `session://` with a specific host if that avoids conflicts, or even a separate scheme that works across the considerations for what's allowed on Desktop for example. Thanks for the PR @privatgt

**Comment by KeeJef** [comment]
I'm going to close this one, as this is going to require cross platform coordination and changes in various locations, which i think are not fully addressed by this PR, we do want to to support deeplinking we just need to get all platforms onboard to the same scheme

**Pull request closed without merging**